### PR TITLE
Warning if use `!top` with chat ID in groups, handling chat not found exception

### LINKED ISSUE #100: !top with notcorrect id without errors in groups

### PR DESCRIPTION
**Pull request by Desiders**
Close #100 and some improvements

### CODE REVIEW

File: app/handlers/karma.py [hunk 1]
```diff
@@ -46,6 +52,11 @@ async def get_top(
     chat_repo: ChatRepo,
     user_repo: UserRepo,
 ):
+    parts = message.text.split(maxsplit=1)
+    if len(parts) > 1:
+        return await message.reply(
+            "Просмотр топа другого чата возможен только в личных сообщениях с ботом"
+        )
```
Comment (bomzheg) [inline app/handlers/karma.py:59]: let's remove this error message after 60 second. and message with command from user too

**Review by bomzheg — CHANGES_REQUESTED** [review]

File: app/handlers/karma.py [hunk 2]
```diff
@@ -45,7 +51,20 @@ async def get_top(
     user: User,
     chat_repo: ChatRepo,
     user_repo: UserRepo,
+    config: Config,
 ):
+    parts = message.text.split(maxsplit=1)
+    if len(parts) > 1:
+        msg = await message.reply(
```
Comment (Lamroy95) [inline app/handlers/karma.py:58]: Naming please. For example `bot_message` or `bot_reply`

File: app/handlers/karma.py [hunk 3]
```diff
@@ -45,7 +51,20 @@ async def get_top(
...
+            "Просмотр топа другого чата возможен только в личных сообщениях с ботом"
+        )
+
+        asyncio.create_task(delete_message(msg, config.time_to_remove_temp_messages))
+        asyncio.create_task(
+            delete_message(message, config.time_to_remove_temp_messages)
+        )
```
Comment (Lamroy95) [inline app/handlers/karma.py:65]: Use `services.remove_message.cleanup_command_dialog` function:
```python
asyncio.create_task(cleanup_command_dialog(bot_message, delete_bot_reply=True, delay=...)
```

**Review by Lamroy95 — CHANGES_REQUESTED** [review]

**Review by Lamroy95 — APPROVED** [review]

File: app/handlers/karma.py [hunk 4]
```diff
@@ -45,7 +51,23 @@ async def get_top(
...
+    if len(parts) > 1:
+        bot_reply = await message.reply(
+            "Просмотр топа другого чата возможен только в личных сообщениях с ботом"
+        )
+
+        asyncio.create_task(
+            cleanup_command_dialog(
+                bot_message=bot_reply,
+                delete_bot_reply=True,
+                delay=config.time_to_remove_temp_messages,
```
Comment (Lamroy95) [inline app/handlers/karma.py:66]: Do we have that key in config?

File: app/handlers/karma.py [hunk 5]
```diff
@@ -60,12 +82,18 @@ async def get_me(message: types.Message, chat: Chat, user: User, config: Config)
...
         disable_web_page_preview=True,
     )
-    asyncio.create_task(delete_message(msg, config.time_to_remove_temp_messages))
-    asyncio.create_task(delete_message(message, config.time_to_remove_temp_messages))
+
+    asyncio.create_task(
+        cleanup_command_dialog(
+            bot_message=bot_reply,
+            delete_bot_reply=True,
+            delay=config.time_to_remove_temp_messages,
```
Comment (Lamroy95) [inline app/handlers/karma.py:94]: Do we have that key in config?

**Review by Lamroy95 — CHANGES_REQUESTED** [review]

File: app/handlers/karma.py (hunk 4 above)
Comment (Desiders) [reply app/handlers/karma.py:66]: https://github.com/bomzheg/KarmaBot/blob/master/app/models/config/__init__.py#L27

File: app/handlers/karma.py (hunk 5 above)
Comment (Desiders) [reply app/handlers/karma.py:94]: https://github.com/bomzheg/KarmaBot/blob/master/app/models/config/__init__.py#L27

**Review by Lamroy95 — APPROVED** [review]

**Pull request merged**